awx/main/utils/ansible.py

```diff
@@ -97,3 +98,14 @@ def could_be_inventory(project_path, dir_path, filename):
     except IOError:
         return None
     return inventory_rel_path
+
+
+def read_ansible_config(project_path, variables_of_interest):
+    fnames = [ os.path.join(project_path, 'ansible.cfg'), '/etc/ansible/ansible.cfg']
```

Comment:
unit tests are failing on the case where `project_path` is none.

https://github.com/ansible/awx/blob/b5fa1606bd14bc96d6f884dfee15bd38d06d8441/awx/main/models/projects.py#L190-L195

This would be an uncommon situation, but one that can happen in practice. The local_path is filled out in some logic around how the project is saved, and I suspect might not be there if it never completed an update, or some weird variation of that.

Anyway, if given a null project_path, then just use

```python
fnames = ['/etc/ansible/ansible.cfg']
```

Let something else in the task logic error instead of this.